tests/integration/test_actor_scrapy.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .conftest import MakeActorFunction, RunActorFunction
+
+from ._utils import read_file
+
+
+async def test_actor_scrapy_title_spider(
+    make_actor: MakeActorFunction,
+    run_actor: RunActorFunction,
+) -> None:
+    actor_source_files = {
+        'src/__init__.py': read_file('docs/02_guides/code/scrapy_project/src/__init__.py'),
```

Comment:
Is the helper function really necessary? I think that `path = Path('cs/02_guides/code/scrapy_project'); (path / 'src/__main__.py').read_text()` would be similarly long and easier to understand.